src/Windowing/Silk.NET.GLFW/GlfwProvider.cs

```diff
@@ -27,8 +27,7 @@ static unsafe GlfwProvider()
                     if (!glfw.Init())
                     {
                         var code = glfw.GetError(out byte* pDesc);
-                        // len = strnlen(pDesc, 4096);
-                        var len = new ReadOnlySpan<byte>(pDesc, 4096).IndexOf((byte)0);
+                        var len = SpanHelpers.IndexOf(ref *pDesc, (byte)'\0', int.MaxValue);
```

Comment:
```suggestion
                        var len = new ReadOnlySpan<byte>(pDesc, int.MaxValue).IndexOf((byte)'\0');
```

Reply:
Yeah my bad, was looking at source.dot.net and it didn't tell me the accessibility of that class. Commit that and I'll merge :D